pylint/checkers/exceptions.py

```diff
@@ -518,8 +528,6 @@ def visit_tryexcept(self, node: nodes.TryExcept) -> None:
                     continue
 
                 for part, exception in exceptions:
```

Comment:
This is unnecessary since `exceptions` is a `SuccessfulInferenceResult` and therefore can't be `Uninferable`.

Reply:
I suppose we'll be able to check that with the coverage.

Reply:
See https://coveralls.io/builds/50914174/source?filename=pylint%2Fcheckers%2Fexceptions.py#L522